release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -95,6 +95,14 @@ For more information about enabling a feature set during installation, see xref:
 [id="ocp-4-12-post-installation"]
 === Post-installation configuration
 
+==== {product-title} with multi-architecture compute machines (Technology Preview)
+
+{product-title} {product-version} with multi-architecture compute machines now supports manifest listed images on image streams. For more information about manifest list images, see xref:../post_installation_configuration/multi-architecture-configuration.adoc[Configuring multi-architecture compute machines on an {product-title} cluster]. 
+
+On a cluster with multi-architecture compute machines, you can now override the node affinity in the Operators `Subscription` object to schedule pods on nodes with architectures that the Operator supports. For more information, see 
```

Comment:
Operators - should this be Operator's or Operator (singular)?

Reply:
Ah ha, in this context yeah its `Operator's`. Thank you!